src/vs/workbench/api/common/extHostTypes.ts

```diff
@@ -740,6 +740,12 @@ export class SnippetString {
 		return this;
 	}
 
+	appendChoice(values: string[], number?: number): SnippetString {
+		this.appendPlaceholder(values.toString(), number);
+		this.value = this.value.replace(/:/g, '|').replace(/}/g, '|}');
```

Comment:
This will do too much I believe - e.g add a placeholder then choice and this regex replaces both occurrences of `:`

Reply:
Thank you! I fixed not to unexpectedly change `this.value`.